### DISCUSSION TITLE
Populate values from one screen to the next screen dynamically

### DISCUSSION THREAD

**Discussion by DamolaGit** [Budibase/budibase#13888] [Help] [answered]
I am having trouble figuring out how to have the values of one row from a table populate another screen when you click on said row.

So I have this table 
![List screen shot](https://github.com/Budibase/budibase/assets/69820192/baa05bad-fc97-4adc-9ae0-21b69f9306e1)

and when I click on one of the rows for example the first row. How do i make sure that the row i clicked on the values of that row are populate into the next screen 
![IncScreenshot](https://github.com/Budibase/budibase/assets/69820192/582a34fa-2cb4-49ab-a782-466924b32248)

I would want that number under offenses to state the INCIDENTNUMBER of the row I clicked on in the table screenshot 
I think I has to do with creating certain bindings between each screen. 

Any help would be appreciated

**Comment by mikesealey** [comment, marked as answer]
Hey @DamolaGit 

This is a question that has come up before. I wrote out an example for another user here in https://github.com/Budibase/budibase/discussions/13796, you'll almost certainly find your answers there.

The tl;dr is that the screen you want to populate should have a URL parameter that accepts the ID of the row you want to display, and when you click the row in your table it navigates to a screen at that URL.

One you've had chance to look at the example posted above, if you have any further questions I'd be happy to answer them here.

**Reply by DamolaGit** [reply]
Hi @mikesealey 

I have gone through the example and tried to replicate it to the best of my ability but I am still running into problems. 

As shown in this screen shot i set the binding onto the table for the first screen
![Table List Binding](https://github.com/Budibase/budibase/assets/69820192/e5a526e7-11b4-4879-b498-8cb03549388c)


And i have set the Row ID for the form as {{ URL.id }} which the data source is the table of the first screen shot.
![Form Budi](https://github.com/Budibase/budibase/assets/69820192/40a1aa4f-0972-4c16-a288-e76e54af068a)


When testing it in the preview it and clicking on a row in the table the form does not appear is there something i did wrong 
![List ](https://github.com/Budibase/budibase/assets/69820192/b7ff1467-a43d-46da-a641-2bb1ebe18a8d)

![Form Preview ](https://github.com/Budibase/budibase/assets/69820192/3a61ea55-8437-4cd0-95fd-ab822206802d)

It states that "We couldn't find a row to display". 

Any help you can provide would be great.

**Reply by mikesealey** [reply]
![image](https://github.com/Budibase/budibase/assets/39269380/9cb4ddbc-566d-4871-97ad-2d9b427ca5bf)

That destination should be more like
`/incident2/{{Incident List Data Provider.INCIDENT_BUDI_T._id}}`

the `/:id` is the variable that you're passing using the `{{``}}` handlebars. The `:` colon tells your browser that it's a variable, and to look out for the value stored in that variable. 

In the example I shared above, the destination is `/employees/{{ New Repeater Block.Employees._id }}`
![image](https://github.com/Budibase/budibase/assets/39269380/43d433c8-219f-4f9e-9d59-c225cdcdaa73)

And the screen URL is `/employees/:id`
![image](https://github.com/Budibase/budibase/assets/39269380/6f4407f5-278a-420d-ab2c-ad9beec498b4)

**Reply by DamolaGit** [reply]
I made the changes but now once I click on a row on the table screen the page does not navigate to the form screen. The page just stays on the table screen is there still something I'm doing wrong?

![List2](https://github.com/Budibase/budibase/assets/69820192/c56de581-b85c-4a9d-bd72-517fae006521)


Would you set the table URL as /incident2/:id

**Reply by DamolaGit** [reply]
Also the screen name for the Table is /list and the screen name for the form is /incident2/:id.

**Reply by DamolaGit** [reply]
![Route List](https://github.com/Budibase/budibase/assets/69820192/4f699feb-fced-4c58-aa19-1ad5174704e2)

![incident2 route](https://github.com/Budibase/budibase/assets/69820192/d9a09c47-4474-4df9-9355-1dd6ecee31b5)
